modeling-cmds/src/def_enum.rs

```diff
@@ -1016,7 +1016,8 @@ define_modeling_cmd_enum! {
             /// Which objects to fit camera to; if empty, fit to all non-default objects. Defaults to empty vector.
             #[serde(default = "default_uuid_vector")]
             pub object_ids: Vec<Uuid>,
-            /// How much to pad the view frame by.
+            /// How much to pad the view frame by, as a fraction of the object(s) bounding box size.
```

Comment:
Can this be positive or negative? If I give negative padding, will it crop out some of the view of the object?

Reply:
good point. updated the docs accordingly!
            /// Negative padding will crop the view of the object proportionally.